docs/asciidoc/modules/ROOT/pages/ml/bedrock.adoc

```diff
@@ -0,0 +1,225 @@
+In case of all procedures, except the `bedrock.list`, the default `endpoint` is `"https://bedrock-runtime.us-east-1.amazonaws.com/model/<modelID>/invoke"`.
+The `<modelID>` part must be configured if we use the `ml.bedrock.custom` procedure,
+while with the `bedrock.jurassic`, `bedrock.anthropic.claude`, `bedrock.titan.embed`, `bedrock.stability` ones,
+has a default value of "ai21.j2-ultra-v1", "amazon.titan-embed-text-v1", "anthropic.claude-v2" and "stability.stable-diffusion-xl-v0" respectively.
+
+
+
+== Authentication settings
+
+To authenticate to bedrock services, we can set in the `apoc.conf` file the following entries.
+
+.apoc.conf
+[source,properties]
+----
+apoc.aws.key.id=<AWS Key ID>
+apoc.aws.secret.key=<AWS Secret Access Key>
+----
+
+Alternatively we can set them as `$config` parameters, i.e.: `{keyId: '<AWS Key ID>', secretKey:'<AWS Secret Access Key>'}`.
+
+Or also, we can put an Authorization header, by using the `header` parameter, 
+i.e. `{header: {Authorization: 'AWS4-HMAC-SHA256 <CredentialAndSignature..>',  ...other entries...} }`.
+
+Note that the default `Content-Type: application/json` and the `Accept: \*/*` header entries,
+are always passed to the http request, unless overridden via the config `header`.
+
+
+In the following examples, 
+we assume that we set Key id and Secret Access Key via `apoc.conf`.
+
+== Usage Examples
+
+=== Ad-hoc model procedure
+
+.apoc.ml.bedrock.jurassic
+[source,cypher]
+----
+CALL apoc.ml.bedrock.jurassic({prompt: "Review: Extremely old cabinets",
+            maxTokens: 50,
+            temperature: 0,
+            topP: 1.0})
+----
+
+.Results
+[opts="header",cols="3"]
+|===
+| id	| promptTokens |	completions
+| 1234 | [
+{
+"textRange": {
+"start": 0,
+"end": 6
+},
+"topTokens": null,
+"generatedToken": {
+"token": "▁Review",
+"raw_logprob": -7.870129585266113,
+"logprob": -7.870129585266113
+}
+}
+,
+{ ....
+| [
+{
+"data": {
+"text": "
+These cabinets are very old and outdated. They are made of wood and have a simple, classic design. The cabinets are in good condition, but they could use some updating.",
+"tokens": [
+{ ...
+|===
+
+
+We can also change the modelId, for example by using the `"ai21.j2-mid-v1"` one, which return the same response values.
+[source,cypher]
+----
+CALL apoc.ml.bedrock.jurassic(<BODY>, {modelId: "ai21.j2-mid-v1"})
+----
+
+
+
+.apoc.ml.bedrock.anthropic.claude
+[source,cypher]
+----
+CALL apoc.ml.bedrock.anthropic.claude({
+                prompt: "\n\nHuman: Hello world\n\nAssistant:",
+            max_tokens_to_sample:  300,
+            temperature: 0.5,
+            top_k: 250,
+            top_p: 1,
+            stop_sequences: ["\\n\\nHuman:"],
+            anthropic_version: "bedrock-2023-05-31"
+})
+----
+
+
+.Results
+[opts="header",cols="2"]
+|===
+|  completion| 	stopReason
+| " Hello!"	| "stop_sequence"
+|===
+
+
+.apoc.ml.bedrock.titan.embed
+[source,cypher]
+----
+CALL apoc.ml.bedrock.titan.embed({inputText: "Hello World"})
```

Comment:
should be `apoc.ml.bedrock.embedding(text-or-list-of-texts, {config with optional model and a default})`